modules/evaluation/src/main/Assessible.scala

```diff
@@ -47,7 +47,7 @@ case class Assessible(analysed: Analysed) {
     highestChunkBlurs(color) >= 7
 
   def highlyConsistentMoveTimes(color: Color): Boolean =
-    if (game.perfType != lila.rating.PerfType.UltraBullet)
+    if (game.perfType != lila.rating.PerfType.UltraBullet && game.clock.map(_.estimateTotalSeconds).getOrElse(0) != 40)
```

Comment:
This is still hard coding a time control to exclude, feels fragile and likely to need continued tweaks.

what about changing the whole condition to
```if (game.clock.forall(_.estimateTotalSeconds > 40))```

Reply:
I think I'd like to keep it for hyperbullet though, which would have 30 as total estimation.

Reply:
I am questioning the idea that 15s+0 and 0+1 are junk but 30s+0 is useful

Reply:
unless you have strong evidence backing why hyperbullet doesn't have false flags, we should assume that faster TCs have more false detection, and pick a cutoff point based on estimateTotalSeconds instead of hand picking which TCs we've seen to be problematic

Reply:
or you could adjust the algorithm to exclude move segments where all the moves are fast.  but I don't like the hand picking of time controls

Reply:
Fair point. I did your cutoff suggestion. On a related note I have no idea how useful 3/4+0 games are here but if they're problematic then we can raise the cutoff.

Reply:
probably doesn't matter about 3/4+0 because they're very rare.  I've seen 30s+0 get falsely detected as consistent though.